facia-press/app/frontpress/FapiFrontPress.scala

```diff
@@ -80,7 +80,94 @@ object EmbedJsonHtml {
   implicit val format = Json.format[EmbedJsonHtml]
 }
 
-trait FapiFrontPress extends Logging {
+case class EmailFrontPath(path: String, edition: String)
+
+object EmailFrontPath {
+  def fromPath(path: String): Option[EmailFrontPath] = path match {
+    case "email/uk/daily" => Some(EmailFrontPath(path, "uk"))
+    case "email/us/daily" => Some(EmailFrontPath(path, "us"))
+    case "email/au/daily" => Some(EmailFrontPath(path, "au"))
+    case _ => None
+  }
+}
+
+case class EmailExtraCollections(canonical: PressedCollectionVisibility,
+                                 special: Option[PressedCollectionVisibility],
+                                 breaking: Option[PressedCollectionVisibility])
+
+trait EmailFrontPress extends Logging {
```

Comment:
I've tried to improve the modularity here by extracting the Email front press functionality in an independent trait rather than throwing everything into `FapiFrontPress`